libs/utils/utils.js

```diff
@@ -983,6 +1000,7 @@ async function checkForPageMods() {
 }
 
 async function loadPostLCP(config) {
+  await decoratePlaceholders(document.body.querySelector('header'), config);
```

Comment:
Why is this needed all of a sudden?

Reply:
This is an interesting one. Before we had
```js
export async function loadArea(area = document) {
  ...
  await decoratePlaceholders(area, config);
```

which changed to
```js
async function processSection(section, config, isDoc) {
  ...
  await decoratePlaceholders(section.el, config);
```

I've only seen that the header (breadcrumbs) doesn't get decorated via `processSection` ... we might need the same call for the footer 🤔 or whole document to be sure.

I wanted to see if we can get away with having the more specific call first, as there is a lot more code to process for the whole document rather than just the header

Reply:
We move the beadcrumbs to the header via JS(`decorateDocumentExtras()`>`decorateHeader()`), hence they are part of the header.

I wonder if they _should_ ? Metadata can be rolled out to all documents within a locale already, hence I'm not sure if we'd need to mix placeholders+metadata